README.md

```diff
@@ -6,6 +6,13 @@ web shop.
 
 == Change log ==
 
+** Version 2.0.0
+
+    * Improvements:
+            - Code refactored according to Magento recommended coding standards
```

Comment:
Since no code structure has really been changed and it's mostly about white space I wouldn't call it refactoring.
```suggestion
            - Code formatted to Magento's recommended coding standards
```

Reply:
Updated